<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PawnDialog</class>
 <widget class="QDialog" name="PawnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>QUEEN</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pic/white_queen.png</normaloff>:/pic/white_queen.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BISHOP</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pic/white_bishop.png</normaloff>:/pic/white_bishop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ROOK</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pic/white_rook.png</normaloff>:/pic/white_rook.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_4">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>KNIGHT</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pic/white_knight.png</normaloff>:/pic/white_knight.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
